panic when .local/bin does not exist

Whenever .local/bin doesn't exist, apx will panic with this:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x7033db]

goroutine 1 [running]:
github.com/vanilla-os/apx/v2/core.findExportedBinaries.func1({0x8f3a30, 0x1}, {0x0?, 0x0?}, {0x8f6ac0, 0xc0006204e0})
	/__w/apx/apx/core/subSystem.go:71 +0x9b
io/fs.WalkDir({0x8f6b60, 0xc000534750}, {0x8f3a30, 0x1}, 0xc0004b13a8)
	/__t/go/1.21.11/x64/src/io/fs/walk.go:120 +0x63
github.com/vanilla-os/apx/v2/core.findExportedBinaries({0xc0004c6390, 0xb})
	/__w/apx/apx/core/subSystem.go:70 +0x1a5
github.com/vanilla-os/apx/v2/core.findExported({0xc0004c6390, 0xb}, {0xc00015e07c, 0x7})
	/__w/apx/apx/core/subSystem.go:176 +0x3f
github.com/vanilla-os/apx/v2/core.ListSubSystems(0x0, 0x0?)
	/__w/apx/apx/core/subSystem.go:297 +0x36e
github.com/vanilla-os/apx/v2/cmd.NewRuntimeCommands()
	/__w/apx/apx/cmd/runtime.go:23 +0x26
main.main()
	/__w/apx/apx/main.go:64 +0x91d

```

--- a/core/subSystem.go
+++ b/core/subSystem.go
@@ -64,15 +64,18 @@ func findExportedBinaries(internalName string) map[string]map[string]string {
 		return map[string]map[string]string{}
 	}
 	binPath := filepath.Join(home, ".local", "bin")
+	if _, err := os.Stat(binPath); os.IsNotExist(err) {
+		return map[string]map[string]string{}
+	}
 	binDir := os.DirFS(binPath)
 
 	binaries := map[string]map[string]string{}
 	err = fs.WalkDir(binDir, ".", func(path string, d fs.DirEntry, err error) error {
-		if d.IsDir() {
-			return nil
-		}
 		if err != nil {
 			return err
+		}
+		if d.IsDir() {
+			return nil
 		}
 
 		file, err := os.Open(filepath.Join(binPath, path))

